### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by hakman**
Ref: https://github.com/google/go-tpm/releases/tag/v0.9.0

### CODE REVIEW

File: upup/pkg/fi/cloudup/gce/tpm/gcetpmsigner/tpm_other.go [hunk 1]
```diff
@@ -23,7 +23,7 @@ import (
 	"fmt"
 	"io"
 
-	"github.com/google/go-tpm/tpm2"
+	"github.com/google/go-tpm/legacy/tpm2"
```
Comment (hakman) [inline upup/pkg/fi/cloudup/gce/tpm/gcetpmsigner/tpm_other.go:26]: @justinsb WDYT?

Comment (justinsb) [reply upup/pkg/fi/cloudup/gce/tpm/gcetpmsigner/tpm_other.go:26]: https://github.com/kubernetes/cloud-provider-gcp/blob/e64cf3b0fcb1958cee1fe55d7e30f4573c34bf4e/pkg/tpmattest/tpmattest.go#L11 hasn't yet updated, so I don't think we want to cherry-pick without a long soak time though!

**Pull request merged**